server/sonar-server/src/main/java/org/sonar/server/ws/ServletRequest.java

```diff
@@ -67,4 +67,14 @@ protected InputStream readInputStreamParam(String key) {
     }
     return null;
   }
+
+  @Override
+  public String toString() {
+    StringBuffer url = source.getRequestURL();
+    String query = source.getQueryString();
```

Comment:
Indeed, but as you say POST params can be pretty big, so we'll do nothing more for the moment for POST.